course-mw/course-actors-common/src/main/java/org/sunbird/learner/util/ContentUtil.java

```diff
@@ -129,8 +129,7 @@ public static Map<String, Object> getContent(String courseId, List<String> field
     Map<String, String> headers = new HashMap<>();
     try {
       String fieldsStr = StringUtils.join(fields, ",");
-      String baseContentreadUrl = ProjectUtil.getConfigValue(JsonKey.EKSTEP_BASE_URL) + "/content/v3/read/" + courseId + "?fields=" + fieldsStr;
-      headers.put(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON);
+      String baseContentreadUrl = ProjectUtil.getConfigValue(JsonKey.EKSTEP_BASE_URL) + PropertiesCache.getInstance().getProperty(JsonKey.CONTENT_READ_URL) + courseId + "?fields=" + fieldsStr;      headers.put(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON);
```

Comment:
Correct the white spacing and line alignment here

Reply:
addressed